Unix/doc/diagnose-omi-problems.md

```diff
@@ -0,0 +1,293 @@
+# Diagnosing OMI Problems
+
+At times, OMI itself may experience problems, or OMI providers may
+experience problems. This documentation will help customers collect
+sufficient information for us to either assist in diagnosing problems
+or allow the customer to isolate problems to specific providers, thus
+allowing support staff to quickly route problems to the appropriate
+team for support/resolution.
+
+Note that this document is broken into several sections. Depending on
+the nature of your problem, some sections may not be appropriate for
+you. However, debugging problems definitely has some common themes.
+Thus, common themes are covered first, and we then dealve into
+specific problems.
+
+- [Minimum OMI version](#minimum-omi-version)
+- [Caviats](#caviats)
+- [Enabling logging in OMI](#enabling-logging-in-omi)
+- [Log file entries of interest](#log-file-entries-of-interest)
+- [Required information when reporting problems](#required-information-when-reporting-problems)
+- [Diagnosing memory problems in OMI itself](#diagnosing-memory-problems-in-omi-itself)
+- [Diagnosing OMI Provider Failures](#diagnosing-omi-provider-failures)
+  - [Diagnosing OMI Provider Memory Leaks](#diagnosing-omi-provider-memory-leaks)
+  - [Diagnosing Unexpected OMI Provider aborts](#diagnosing-unexpected-omi-provider-aborts)
+
+### Minimum OMI Version
+
+**NOTE: This document assumes OMI v1.4.1-0 or later.** If you have a
```

Comment:
I know you discuss how to get version number later.  Either move that part up here, or note here that you'll describe how to get version number later.

Reply:
I copied it so it's in both places.